pallet/runtime-transact/src/lib.rs

```diff
@@ -44,23 +44,23 @@ use sp_std::boxed::Box;
 pub use pallet::*;
 
 #[derive(Clone, Eq, PartialEq, RuntimeDebug, Encode, Decode, MaxEncodedLen, TypeInfo)]
-pub enum LcmpEthOrigin {
-	MessageTransact(H160),
+pub enum RuntimeEthOrigin {
```

Comment:
Why use enum here

Reply:
Currently only stake module uses it, it may later contain several variables, such as `StakingTransact(H160)`, `PalletBTransact(H160)`,xxx.